nsxt/resource_nsxt_policy_ipsec_vpn_service_test.go

```diff
@@ -0,0 +1,193 @@
+/* Copyright © 2022 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+var accTestPolicyIPSecVpnServiceCreateAttributes = map[string]string{
+	"display_name":        getAccTestResourceName(),
+	"description":         "terraform created",
+	"locale_service_path": "/infra/tier-0s/test-t0/locale-services/default",
```

Comment:
The T0 gateway needs to be created within the test, since we can't rely on test-t0 to be pre-created on the testbed. We usually just configure the resources the test depends on as part of the template.

Reply:
I've changed the code to use the EdgeCluster and Tier0 template in the test_utils files, and get the locale_service_path from the tier0.